employee-rostering-frontend/src/ui/header/Header.test.tsx

```diff
@@ -20,8 +20,38 @@ import * as React from 'react';
 import Header from './Header';
 
 describe('Header component', () => {
-  it('should render correctly', () => {
-    const header = shallow(<Header />);
+  it('should render correctly on a desktop', () => {
+    const mediaQuery = jest.requireMock('react-responsive');
+    mediaQuery.useMediaQuery.mockReturnValueOnce(true);
+
+    const header = shallow(<Header {...props} />);
+
+    expect(mediaQuery.useMediaQuery).toBeCalledWith({ minWidth: 1400 });
+    expect(toJson(header)).toMatchSnapshot();
+  });
+
+  it('should render correctly on a mobile', () => {
+    const mediaQuery = jest.requireMock('react-responsive');
+    mediaQuery.useMediaQuery.mockReturnValueOnce(false);
+
+    const header = shallow(<Header {...props} />);
+
+    expect(mediaQuery.useMediaQuery).toBeCalledWith({ minWidth: 1400 });
     expect(toJson(header)).toMatchSnapshot();
   });
-});
\ No newline at end of file
+
+  it('logo should go to optaplanner.org', () => {
+    const mediaQuery = jest.requireMock('react-responsive');
+    mediaQuery.useMediaQuery.mockReturnValueOnce(true);
+
+    const header = shallow(<Header {...props} />);
+    const logo = shallow(header.prop("logo"));
+
+    expect(logo.prop("href")).toEqual("https://www.optaplanner.org/");
```

Comment:
I would prefer if the link opened in a new browser tab. I think if you take most web apps, for example OpenShift web console or Gmail, the top-left corner logo usually takes you to the landing page of the app. I don't have a problem with pointing to www.optaplanner.org but as a user, I'd find it annoying that the link would make me _leave_ the app I'm working with (think about URLs in Gmail message - they always open a new tab).

Reply:
Oh, this is the test. Moving my original comment to `Header.tsx`.